src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseAuditEventRepository.java

```diff
@@ -41,10 +42,13 @@ public AuditEvent set(final AuditEvent auditEvent) {
     }
 
     public List<AuditEvent> findByCase(final CaseDetails caseDetails) {
-        final Query query = em.createNamedQuery(CaseAuditEventEntity.FIND_BY_CASE);
-        query.setParameter(CaseAuditEventEntity.CASE_DATA_ID, Long.valueOf(caseDetails.getId()));
+        final List<CaseAuditEventEntity> resultList = em.createNamedQuery(CaseAuditEventEntity.FIND_BY_CASE)
```

Comment:
Could you update existing integration test by asserting data to null ?

Reply:
👍